cmd/wait.go

```diff
@@ -154,7 +160,7 @@ func (w *Waiter) Wait() error {
 		}
 
 		ok, err := w.done(r.Type, r.Id)
-		if err != nil {
+		if err != nil && w.state == "" {
```

Comment:
Please provide input for this line. Without `w.state == ""`, if the state is set, and the stack reaches an error while transitioning, it will still return from the cli. With this check, it'll continue waiting until the `--wait-flag` state agreed upon is reached.